docs/sphinx/manual/creating_hpx_projects.rst

```diff
@@ -254,8 +256,14 @@ This requires that you have already created the target like this:
    target_include_directories(hello_world_component PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
 When you link your library to ``HPX::hpx`` you will be able use |hpx|
```

Comment:
Should we mention that `HPX::hpx` is a cmake target exported by the HPX cmake build system setup and that linking against it in this context means using a `target_link_libraries(<user_target> PRIVATE/PUBLIC HPX::hpx)`?

Reply:
Yes, good idea. I'll at least explicitly say `the CMake target ``HPX::hpx`` `.

Reply:
Updated here and there. `CMake target` is mentioned many times now, including in the section heading ;)